content/articles/2018-08-scales-1-0-0.Rmarkdown

```diff
@@ -0,0 +1,106 @@
+---
+title: "scales 1.0.0"
+author: Dana Seidel
+date: '2018-08-09'
+slug: scales-1-0-0
+description: >
+  scales 1.0.0 is now on CRAN.
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/3kzlCL3rj8A
+  author: Ricardo Gomez Angel
+tags: []
+---
+
+```{r setup, include=FALSE}
+library(scales)
+library(dplyr)
+library(ggplot2)
+
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+```
+
+We're delighted to announce the release of [scales](https://scales.r-lib.org/) 1.0.0. 
+This is a major release with significant changes to the popular formatter
+functions, and added transformations.
+See the [News](https://scales.r-lib.org/news/index.html) for a detailed list of changes.
+
+Install the latest version with:
+```{r, eval = FALSE}
+install.packages("scales")
+```
+
+# New features
```

Comment:
I think you can remove the new features heading.

You need a bulleted list that says what the rest of the post will cover

Reply:
Should also mention that scales now has website!